user_docs/en/changes.t2t

```diff
@@ -8,8 +8,18 @@ What's New in NVDA
 == New Features ==
 - Added a "Quick Start Guide" section to the User Guide. (#13934)
 - Introduced a new command to check the keyboard shortcut of the current focus. (#13960)
-  - Desktop: ``shift+numpad2``
-  - Laptop: ``NVDA+ctrl+shift+.``
+  - Desktop: ``shift+numpad2``.
+  - Laptop: ``NVDA+ctrl+shift+.``.
+  -
+- Introduced new commands to move the review cursor by page where supported by the application.

```

Comment:
@seanbudd I think it's worth calling out the utility of these commands in Windows Terminal as it isn't immediately obvious. Unless you think not for some reason?

Reply:
Isn't a terminal change still blocking this?

I think this kind of extra information is better suited to the release blurb, in-process or in the WinConsole section of the user guide.

Reply:
> Isn't a terminal change still blocking this?

Yes, but it'll almost certainly be in by 2022.4.

Reply:
> WinConsole section of the user guide

Not applicable here, as these are for Terminal (but maybe a far future conhost).

Reply:
I would suggest adding a note to the user guide, and add it to the changelog as well. This should probably wait until the terminal feature is introduced.

Reply:
Perhaps a new section in the user guide then?